Automattic/woocommerce-connect-client
# Basic text area support and debug log tail

Next item for #153 - adds debug log tail and debug on/off toggle

Also tweaks save form to work from store state instead of state mapped to prop

To test:
Navigate to wp-admin > WooCommerce > System Status > WooCommerce Connect
Scroll down to the debug section
See the debug log snippet
Make sure the link to the full log works
Make sure turning logging off and on works
